apiary.apib

```diff
@@ -3063,4 +3074,167 @@ List licenses available to your User / Organization.
+    ]
+
+## Auths [/auths]
+
+These endpoints provide user/device authentication.
+
+
+### Request reset password link to user's email [PATCH /auths/users/reset]
+
+Request to send password reset link to user's email.
+
++ Request (application/json)
+    + Headers
+        Authorization: Bearer <api token>
+    
+    + Body
+
+        {
+            "Email": "user@email.com",
+            "Identifier": "organization_identifier"
+        }
+
++ Response 200 (application/json)
+
+    {
+        "success": true
+    }
+
++ Response 403 (application/json)
+
+        {
+            "Code": "USER_PASSWORD_RESET_ERROR", 
+            "Message": "No such user."
+        }
+
++ Response 403 (application/json)
+
+        {
+            "Code": "USER_RESET_ERROR", 
+            "Message": "Could not reset user password"
+        }
+
+
+### Reset password [PATCH /auths/users]
+
+Reset password.
+
++ Request (application/json)
+    + Headers
+        Authorization: Bearer <api token>
+    
+    + Body
+
+        {
+            "ResetToken": "reset_token",
+            "Password": "password12345"
+        }
+
++ Response 200 (application/json)
+
+    {
+        "success": true
+    }
+
++ Response 403 (application/json)
+
+        {
+            "Code": "INVALID_RESET_TOKEN", 
+            "Message": "Invalid reset token provided. please try resetting password again"
+        }
+
+### Request reset password link to user's phone number [PATCH /auths/users/reset/mobile]
+
+Request to send password reset link to user's phone number.
+
++ Request (application/json)
+    + Headers
+        Authorization: Bearer <api token>
+    
+    + Body
+
+        {
+            "PhoneNumber": "phone_number",
+            "Email": "user_email"
+        }
+
++ Response 200 (application/json)
+
+    {
+        "success": true
+    }
+
++ Response 404 (application/json)
+
+        {
+            "Code": "USER_NOT_FOUND", 
+            "Message": "User not found."
+        }
+
++ Response 403 (application/json)
+
+        {
+            "Code": "USER_RESET_ERROR", 
+            "Message": "Could not reset user password"
+        }
+
+### Request login code [PATCH /auths/users/code]
+
+Request to send login code.
+
++ Request (application/json)
+    + Headers
+        Authorization: Bearer <api token>
+    
+    + Body
+
+        {
+            "PhoneNumber": "phone_number",
+            "Identifier": "organization_identifier"
+        }
+
++ Response 200 (application/json)
+
+    {
+        "success": true
+    }
+
++ Response 401 (application/json)
+
+        {
+            "Code": "LOGIN_METHOD", 
+            "Message": "Invalid login method! Try to login with an Email."
+        }
+
++ Response 404 (application/json)
+
+        {
+            "Code": "INVALID_USER", 
+            "Message": "User not found"
+        }
+
+### Renew [POST /auths/renew]
+
+Renew user/device access and refresh token. When register a license or login with user's credentials, a pair 
```

Comment:
Can you please do the following substitutions

s/When register/When you register/